roles/development/files/bin/make-changelog.py

```diff
@@ -50,42 +53,84 @@ def get_tags():
     cmd = "git tag -l"
     tags = run(cmd)
     tags.sort(key=LooseVersion)
-    tags.append('develop')
+    if 'develop' in run('git branch'):
+        tags.append('develop')
     return list(reversed(tags))
 
 
 def get_commits(start, stop):
-    cmd = "git log {start}...{stop} --pretty=format:'%H %s' --reverse | cat"
+    cmd = "git log {start}...{stop} --pretty=format:'%H %ci %s' --reverse | cat"
     cmd = cmd.format(start=start, stop=stop)
     output = run(cmd)
-    commits = [line.split(None, 1) for line in output]
+    commits = []
+    for line in output:
+        line = line.split(None, 4)
+        commits.append([line[0], ' '.join(line[1:4]), line[4]])
```

Comment:
To make this more readable, could you add an extra line?  Something like this:

```python
line = line.split(None, 4)
foo, bar, baz = line[0], ' '.join(line[1:4]), line[4]
commits.append([foo, bar, baz])
```

Otherwise the reader is left guessing about what those three things are.